src/components/Dashboard/Header/Authentication/Signup.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, shallow } from 'enzyme';
+import { BrowserRouter } from 'react-router-dom';
+import Signup from './Login';
+
+describe('Signup', () => {
+  it('Login should render correctly', () => {
```

Comment:
you can write also tests to check if user can log in with invalid name/email/pass